### PR TITLE
Xfail TestEverflowV4EgressAclEgressMirror for Cisco platform until fully functional

### PR DESCRIPTION
**Pull request by wsycqyz**
### Description of PR

Summary:
Everflow Egress ASIC capability got enabled on Cisco platform, so the Egress sonic-mgmt tests are executed.
Skip until full functionality of Everflow Egress is validated on tested on Cisco side.
Cisco JIRA to track the issue: https://migsonic.atlassian.net/browse/MIGSMSFT-677

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

### CODE REVIEW

**Review by XuChen-MSFT — APPROVED** [review]
All good !

**Comment by kevinskwang** [comment]
is this a new case? if not, why we need to skip since 202405

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -690,9 +690,13 @@ everflow/test_everflow_testbed.py::EverflowIPv4Tests::test_everflow_dscp_with_po
 
 everflow/test_everflow_testbed.py::TestEverflowV4EgressAclEgressMirror:
   skip:
-    reason: "Due to HW resource limitation, need to skip the test on the Mellanox t0-120 setup"
+    reason: "Due to HW resource limitation, need to skip the test on the Mellanox t0-120 setup."
```
Comment (kevinskwang) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:693]: remove this redundant change?

**Comment by vperumal** [comment]
Hi @kevinskwang  - Can we use this PR https://github.com/sonic-net/sonic-mgmt/pull/14964 instead - Abhishek had requested to also skip DSCP policer instead of xfailing it because it was causing the test to fully run and then xfail it.

**Comment by wsycqyz** [comment]
> Hi @kevinskwang - Can we use this PR #14964 instead - Abhishek had requested to also skip DSCP policer instead of xfailing it because it was causing the test to fully run and then xfail it.

Close this PR.
@vperumal

**Pull request closed without merging**